Sources/BraintreePayPal/BTPayPalVaultRequest.swift

```diff
@@ -39,21 +31,21 @@ import BraintreeCore
     /// Initializes a PayPal Vault request
     /// - Parameters:
     ///   - offerCredit: Optional: Offers PayPal Credit if the customer qualifies. Defaults to `false`.
+    ///   - userAuthenticationEmail: Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
```

Comment:
nit: can we move the `userAuthenticationEmail` parameter docstrings right after  `recurringBillingPlanType` as it was before